app/src/js/reducers/providers.js

```diff
@@ -102,12 +102,6 @@ export default createReducer(initialState, {
     state.list.inflight = false;
     state.list.error = false;
   },
-  [PROVIDERS]: (state, action) => {
```

Comment:
Just because I don't know things... why is this removed?

Reply:
because it was duplicated in the code.